package.json

```diff
@@ -1,105 +1,21 @@
 {
```

Comment:
We should add a new root-level readme that summarizes our docs.

Reply:
Looks like @yoshiwarab has implemented this, resolving 🙏